scripts/release/class-api-analyzer.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+import fs from 'node:fs';
+import path from 'node:path';
+import TypeDoc from 'typedoc';
+
+import { error, errorHandler, log, success } from '../helpers/index.js';
+import { ELEMENT_DIST, ELEMENT_SOURCE, generateDocList } from './util.js';
+
+/**
+ * Analyzes class public API from TypeScript, output a JSON file
+ * @returns {void}
+ */
+const handler = async () => {
+  log('Analyzing class API...');
+
+  try {
+    for (const item of generateDocList) {
```

Comment:
There is still room for improvement :)
```suggestion
    for (const { entry, tsconfig } of generateDocList) {
```